docs/content/index.md

```diff
@@ -20,7 +20,8 @@ These environments can be extended, version controlled, and shared, so you can t
     * RAM: 8GB
     * Storage: 256GB
     * [Colima](https://github.com/abiosoft/colima) or [Docker Desktop](https://www.docker.com/products/docker-desktop/)
```

Comment:
Is there a definitive minimum version for Colima? It’d be nice to be able to say “Colima (macOS X–latest) or Docker Desktop (macOS 11 Big Sur or higher)”. This would clarify the confusion about requirements, which is exactly what this section focuses on.

The remaining bullets aren’t especially relevant here:

- What people prefer should probably be a component of whatever we officially recommend, which is still not a requirement.
- Colima and Docker Desktop are both fully supported as far as I know, so the “(fully supported)” parenthetical seems redundant and to apply that perhaps Colima is _not_ fully supported somehow.
- The Mutagen comment is out of place unless it has a direct impact on requirements, and people should be encouraged to install it not in the requirements but on the installation page.

Reply:
Actually, there isn't a minimum requirement for Colima, but there *is* for homebrew, and it would be nearly impossible to install without homebrew, so that's a good point. And... it turns out that Big Sur is also the minimum requirement for homebrew. 

Mutagen is actually a requirement with colima; if you use sshfs with colima then things don't work without mutagen.

Reply:
Ooof. Sounds like there could be a page just for navigating conditional requirements. Maybe it’d be best to list Big Sur as the requirement and add an FAQ like “Can DDEV run on an older Mac” that steps through this?

Or if it turns out nobody’s running older macOS versions and it’s over-serving an edge case, call out Big Sur and make sure the Colima instructions mention Mutagen and move on.

Reply:
I just put Big Sur for both in the end.

DDEV works in a lot lot lot of different environments. It would be easier by far to limit them. But I do love that it works so many different places.

Lando actually requires (and installs during installation process, whether you already have docker or not) its own supported version of Docker Desktop.. and if you go outside that you're on your own.